src/webapp/components/copy-in-user-dialog/CopyInUserDialog.tsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import i18n from "../../../locales";
+import { useAppContext } from "../../contexts/app-context";
+import { User } from "../../../domain/entities/User";
+import { Id } from "../../../domain/entities/Ref";
+import { AccessElements, UpdateStrategy } from "../../../domain/repositories/UserRepository";
+import { ConfirmationDialog, MultiSelector } from "@eyeseetea/d2-ui-components";
+import { useSnackbar } from "@eyeseetea/d2-ui-components";
+import { Toggle } from "material-ui";
+import { Box, makeStyles } from "@material-ui/core";
+import styled from "styled-components";
+import _ from "lodash";
+
+export const CopyInUserDialog: React.FC<CopyInUserDialogProps> = props => {
+    const { d2 } = useAppContext();
+    const { onCancel, onSave, user, visible, usersList } = props;
+
+    const [selectedUsersIds, setSelectedUsersIds] = React.useState<Id[]>([]);
+    const [updateStrategy, setUpdateStrategy] = React.useState<UpdateStrategy>("merge");
+    const [accessElements, setAccessElements] = React.useState<AccessElements>({
+        userGroups: false,
+        userRoles: false,
+        dataViewOrganisationUnits: false,
+        organisationUnits: false,
+    });
+
+    const snackbar = useSnackbar();
+
+    // Overide TextInput width in MultiSelector
+    const useStyles = makeStyles(() => ({
+        searchFieldOverride: {
+            width: "initial",
+        },
+    }));
+
+    const isReplaceStrategy = updateStrategy === "replace";
+    const strategyLabel = isReplaceStrategy ? i18n.t("Replace") : i18n.t("Merge");
+    const copyInUserTitle = i18n.t("Copy in user: {{user}}", {
+        user: user.username,
```

Comment:
The library `i18next-scanner` use the colon `:` as a separator to find the namespace translation. If you run `yarn localize` you will see the following error:

```bash
i18next-scanner: "Copy in user" does not exist in the namespaces (["translation"]): key=" {{user}}", options={}
```

Since we have the `:` in the text let's add the `nsSeparator: false` parameter to avoid the error.

```
i18n.t("Copy in user: {{user}}", {
        user: user.username,
        nseparator: false,        
    });
```